pkgs/applications/networking/browsers/webmacs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, qtbase
+, herbstluftwm
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webmacs";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner  = "parkouss";
+    repo   = "webmacs";
+    rev    = "d692ae8c2bf82cce854f34cb3d762932fe7fdc8b";
+    fetchSubmodules = true;
+    sha256 = "1hzb9341hybgrqcy1w20hshm6xaiby4wbjpjkigf4zq389407368";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    python3Packages.pyqt5_with_qtwebkit
```

Comment:
```suggestion
    pyqtwebengine
    setuptools
```

Reply:
Bingo! `setuptools` was missing. Thanks!

I'll try to clear up the huge exploratory mess I have locally, and submit a clean version.

Reply:
Sorry, I hid this change a bit

Reply:
> I'll try to clear up the huge exploratory mess I have locally, and submit a clean version.

Done.